swervelib/MotorType.java

```diff
@@ -1,5 +1,5 @@
 package frc.team670.mustanglib.swervelib;
 
 public enum MotorType {
-    FALCON, NEO;
+    FALCON, NEO, KRAKEN;
```

Comment:
should probably call it KRAKENX60 or KRAKEN_X60 - their naming scheme clearly implies there'll be more kraken family motors